File: src/main/java/com/project/food_delivery/dtos/AddressDto.java
package com.project.food_delivery.dtos;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;

@Data
public class AddressDto {
    @Schema(description = "Id", example = "0")
    private Integer id;

    @Schema(description = "Name", example = "Ivano-Frankivsk")
    private String city;

    @Schema(description = "Street", example = "Ivan Mykolaychuk street")
    private String street;

    @Schema(description = "Building number", example = "20")
    private String buildingNum;
}

File: front-end/src/app/common/address-dto.ts
export class AddressDto {
    public id: number;

    public city: string;

    public street: string;

    public buildingNum: string;

    constructor(id: number, city: string, street: string, buildingNum: string) {
        this.id = id;
        this.city = city;
        this.street = street;
        this.buildingNum = buildingNum;
    }
}
